pychromecast/socket_client.py

```diff
@@ -860,24 +882,21 @@ def send_message(
         # If channel is not open yet, connect to it.
         self._ensure_channel_connected(destination_id)
 
-        request_id = None
         if not no_add_request_id:
             request_id = self._gen_request_id()
             data[REQUEST_ID] = request_id
 
         if inc_session_id:
             data[SESSION_ID] = self.session_id
 
-        msg = cast_channel_pb2.CastMessage()  # pylint: disable=no-member
+        msg = CastMessage()
 
         msg.protocol_version = msg.CASTV2_1_0
         msg.source_id = self.source_id
         msg.destination_id = destination_id
-        msg.payload_type = (
-            cast_channel_pb2.CastMessage.STRING  # pylint: disable=no-member
-        )
+        msg.payload_type = CastMessage.STRING
         msg.namespace = namespace
-        msg.payload_utf8 = _json_to_payload(data)
+        msg.payload_utf8 = _json_to_payload(data)  # type: ignore[assignment]
```

Comment:
The existing code works, but violates the typing of `CastMessage` which claims `CastMessage.payload_utf8` is `str`.